lib/turn.rb

```diff
@@ -0,0 +1,61 @@
+require 'pry'
+require_relative './card'
+require_relative './deck'
+require_relative './player'
```

Comment:
Inside the `Turn` class we never create a `Card`, `Deck`, or `Player` object therefore no need for the `require`'s.